src/users/containers/invite-user-success/invite-user-success.component.html

```diff
@@ -1,8 +1,23 @@
 <div class="hmcts-width-container">
     <main role="main" class="govuk-main-wrapper">
-        <app-success-notification [notification]="'New user invited'"></app-success-notification>
-        <p class="govuk-body">
-            <span text><a [routerLink]="['/users/invite-user']">Invite another User</a></span>
-        </p>
-    </main>
+        <div class="govuk-grid-row">
+            <div class="govuk-grid-column-two-thirds">      
+                <div class="govuk-panel govuk-panel--confirmation">
+                    <h1 class="govuk-panel__title">
+                        You 've invited
```

Comment:
the is a space in "You 've" that should not be there it should be "You have" in any case